tox.ini

```diff
@@ -6,3 +6,13 @@ deps = pytest
 install_command = pip install -e .[test] --find-links {toxinidir}/3rdparty {opts} {packages}
 # TODO(xujyan): Fix up 'test_scheduler_runs'.
 commands = py.test -v -k 'not test_scheduler_runs' tests {posargs:}
+
+[testenv:style]
+deps =
+  twitter.common.app==0.3.0
+  twitter.common.collections==0.3.0
+  twitter.common.lang==0.3.0
+  twitter.common.log==0.3.0
+  twitter.checkstyle==0.1.0
```

Comment:
should only need the twitter.checkstyle dep, see https://github.com/pantsbuild/pex/blob/master/tox.ini#L72